### PR TITLE
Add Link Flair Icons

### PR DESCRIPTION
**Pull request by Thunderclaww**
Archive Heroes 2.0 logo

### CODE REVIEW

**Review by yuna9 — COMMENTED** [review]
What's the reason for the extra space on the sides of `images/link-flair.png`?

File: link_flair.scss [hunk 1]
```diff
@@ -69,6 +69,58 @@
...
+.linkflair-esports .thumbnail.self {
+  background: url(%%link-flair%%) 0 -56px
+}
+
+.linkflair-bug .thumbnail.self {
+  background: url(%%link-flair%%) 0 -112px
+}
+
+.linkflair-suggestion .thumbnail.self {
+  background: url(%%link-flair%%) 0 -168
```
Comment (yuna9) [inline link_flair.scss]: missing `px`

**Review by yuna9 — CHANGES_REQUESTED** [review]
small nits

File: link_flair.scss (hunk 1 above)
Comment (Thunderclaww) [reply link_flair.scss]: RIP

**Review by yuna9 — APPROVED** [review]

**Pull request merged**